composer.lock

```diff
@@ -5587,12 +5587,12 @@
             "source": {
```

Comment:
why was this file updated.

Reply:
This is the update of the ps_apiresource module